src/operators/concat-static.ts

```diff
@@ -15,7 +15,6 @@ export default function concat<R>(...observables: Array<Observable<any> | Schedu
   let args = <any[]>observables;
   if (typeof (args[observables.length - 1]).schedule === 'function') {
     scheduler = args.pop();
-    args.push(1, scheduler);
```

Comment:
Seems obviously bit weird. @blesh , was there some sort of reason? any kind of optimizations, or js runtime specific issues maybe?